src/app/shared/services/search/search.service.ts

```diff
@@ -24,7 +24,7 @@ import { SearchState } from '../../state/search/search.state';
  */
 function createSearchCriteriaDescription({ ageRange: [minAge, maxAge], gender }: SearchStateModel): string {
   let ageDescription = `Age ${defaultTo(minAge, 18)}-${defaultTo(maxAge, '89+')}`;
-  if (minAge === maxAge || minAge === 89 || maxAge === 18) {
+  if (minAge === maxAge || minAge === 90 || maxAge === 18) {
```

Comment:
minAge >= 89

Reply:
We should also aim to remove these magic numbers and instead take it from configuration somewhere.

Reply:
Definitely, that was a mistake on my part for not doing it from the beginning